src/main/resources/messages.properties

```diff
@@ -448,6 +448,17 @@ applicant-applying.ssn=What is your Social Security Number (SSN)?
 applicant-applying.ssn-help=Only add yours if you would like to get benefits. Otherwise, click "Next".
 applicant-applying.ssn-why=Why do we ask for SSNs?
 
+# Why do we ask for SSNs
+ssn-why.title=Why we ask for Social Security Numbers (SSNs)?
+ssn-why.who=Who needs to give their SSN?
+ssn-why.who-1=Only add SSNs for people in your household who want to get benefits.
+ssn-why.who-2=You don't need to add SSNs for people who do not want to apply for benefits.
+ssn-why.why=Why do I need a SSN to get benefits?
+ssn-why.why-1=The law says everyone who gets SNAP food benefits needs to give their SSN.
+ssn-why.why-2=If you or someone else wants to get benefits without a SSN, it needs to be a good reason (such as being a newborn).
+ssn-why.why-3=The name of the law that gives us permission to ask for SSNs is the Food and Nutrition Act of 2008, as amended, 7 U.S.C. 2011?2036.
+ssn-why.what=What will you do with my SSN after I add it
```

Comment:
Noting this last section is not done yet, as it's pending active conversation happening here:
https://cfastaff.slack.com/archives/C06E3AQ76FK/p1708548463969789

Shalaka said we can submit as is and add the copy needed in a follow up ticket later.